playbooks/ORCHESTRATION.md

```diff
@@ -127,9 +127,70 @@ Manually check for:
 
 <!-- TODO: Add playbook for disaster/recovery steps. -->
 
-### Hardware Key Signing
+## Step 3: Add delegation (Optional)
+
+This step will add a delegated role to the top-level targets that is
+controlled by an external GitHub repository. Coordinate with the
+delegation keyholder to run the `add-delegation` (see
+https://github.com/sigstore/root-signing/blob/main/cmd/tuf/app/add-delegation.go).
+When creating the delegation with the command, a `target-meta`
+file has to be provided that lists the targets, similar to adding
+the top level targets.
```

Comment:
It would be useful to provide a simple example of the `target-meta` file, or link to an example in the repo.

Reply:
There is one, in the section above: https://github.com/kommendorkapten/root-signing/blob/delegation_documentaton/playbooks/ORCHESTRATION.md#targets-and-delegation-configuration

I'll update to include a link to that.